src/components/TodoTask.jsx

```diff
@@ -0,0 +1,82 @@
+import React, {useState} from 'react';
+import { FaRegCheckCircle } from "react-icons/fa";
+import { RiDeleteBinLine } from "react-icons/ri";
+import { FaRegEdit } from "react-icons/fa";
+
+export default function TodoTask({text, deleteTask, editTask, id}) {
+    
+    const [isEditing, setEditing] = useState(false);
+    const [newInputText, setNewInputText] = useState("");
+    const [isDone, setIsDone] = useState(false);
+
+    function handleChange(event) {
+        setNewInputText(event.target.value);
+    }
+
+    function handleSubmit (event) {
+        event.preventDefault();
+    };
+
+    function handleClick () {
+        setIsDone(!isDone);
+    }
+
+    const editingTemplate = (
+        <div className="edit-form">
+            <form onSubmit={handleSubmit}>
+                <input 
+                    onChange={handleChange}
+                    type="text" 
+                    value={newInputText}
+                />
+                <button 
+                    className="edit-button"
+                    type="submit"
+                    onClick={() => {
+                        if (newInputText !== "") {
+                            editTask(id, newInputText);
+                            setEditing(false);
+                        }
+                    }}
+                >
+                <span>Edit</span>
+                </button>
+            </form>
+        </div>
+    );
+
+    const viewTemplate = (
+        <div className="task-container">
+            <div style={{textDecoration: isDone ? "line-through" : "none"}}>
```

Comment:
Ce design, mie mi se pare cam greu de văzut linia de tăiere. Ai putea să îi dai altă culoare.

```suggestion
            <div style={{textDecoration: isDone ? "line-through red" : "none"}}>
```